src/Controls/Maps/src/Map.cs

```diff
@@ -111,19 +113,14 @@ public bool MoveToLastRegionOnLayoutChange
 			set { SetValue(MoveToLastRegionOnLayoutChangeProperty, value); }
 		}
 
-		public IList<MapElement> MapElements => _mapElements;
+		public IList<IMapElement> MapElements => _mapElements;
 
-		public event EventHandler<MapClickedEventArgs> MapClicked;
+		public event EventHandler<MapClickedEventArgs>? MapClicked;
 
-		[EditorBrowsable(EditorBrowsableState.Never)]
-		public void SendMapClicked(Position position) => MapClicked?.Invoke(this, new MapClickedEventArgs(position));
-
-		[EditorBrowsable(EditorBrowsableState.Never)]
-		public void SetVisibleRegion(MapSpan value) => VisibleRegion = value;
-		public MapSpan VisibleRegion
+		public MapSpan? VisibleRegion
 		{
 			get { return _visibleRegion; }
-			internal set
+			set
```

Comment:
Should `VisibleRegion` have a public setter? 

It doesn't seem like `VisibleRegion` propagates down to the handlers so it seems like we shouldn't expose this to the user.

If we're going to use `MoveToRegion` as the mechanism for a user to change the region then I think we'd make this property readonly and just use interfaces to expose the the `VisibleRegion` setter for the handler to communicate through.

This feels like we should copy the `ScrollView` pattern a bit here where the `Horizontal` and `Vertical` offsets aren't exposed as public properties they are just settable through the interface

Reply:
Ok gonna check ScrollView

Reply:
Fixed @PureWeen

Reply:
Oh I think I unfixed this with https://github.com/dotnet/maui/pull/7886/commits/c38b7d96603085eab35212b5ad15b2ff06f3048f 

Sorry! A test was complaining that it couldn't find the setter now, so I added it... I _think_ this should fix the build for now, but before we merge and sign off on it, please recheck this bit of code 😄